backend/scripts/k8s_vector_cleanup.py

```
@@ -0,0 +1,228 @@
+        logger.info(f"⏰ Web search cleanup: {web_search_expiry_days} days retention")
+        logger.info(
+            f"⏰ Collection cleanup: {collection_cleanup_days} days retention (prevents uncontrolled growth)"
+        )
+
+        logger.info("✓ Environment validation completed")
+
+        # Import cleanup functions
+        from open_webui.routers.retrieval import (
+            cleanup_orphaned_vectors,
+            cleanup_expired_web_searches,
+            cleanup_orphaned_chat_files,
```

Ruff (0.12.2)

`open_webui.routers.retrieval.cleanup_orphaned_chat_files` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)